### ISSUE TITLE
Crashing with Optifine / Optifbric

### ISSUE THREAD

**Issue by lollingtonbear** [Zergatul/freecam#3]
Hi,

Using MultiMC to run MC client (MC 1.19).

Looks like clashing mods are Optifine and Freecam. I've tried both with other mods and they all work fine so this is isolated to just these 2.

MOD versions being used are listed below. 
```
fabric-api-0.57.0+1.19
optifabric-1.13.5
OptiFine_1.19_HD_U_H8_MOD
zergatul.freecam-1.2.0-fabric-1.19
```

Crash dump:
```
MultiMC version: 0.6.16-3431


Launched instance in online mode

authserver.mojang.com resolves to:
    [18.165.195.40, 18.165.195.47, 18.165.195.37, 18.165.195.49]

session.minecraft.net resolves to:
    [52.45.69.223, 3.229.83.86, 52.7.214.17]

textures.minecraft.net resolves to:
    [18.165.201.70, 18.165.201.65, 18.165.201.12, 18.165.201.77]

api.mojang.com resolves to:
    [18.172.153.32, 18.172.153.104, 18.172.153.35, 18.172.153.63]


Minecraft folder is:
E:/Minecraft/MultiMC/instances/1.19-Fabric/.minecraft


Java path is:
C:/Program Files/Java/jdk-17.0.1/bin/javaw.exe


Checking Java version...
Java is version 17.0.1, using 64-bit architecture, from Oracle Corporation.


Main Class:
  net.fabricmc.loader.impl.launch.knot.KnotClient

Native path:
  E:/Minecraft/MultiMC/instances/1.19-Fabric/natives

Traits:
traits FirstThreadOnMacOS
traits XR:Initial

Libraries:
  E:/Minecraft/MultiMC/libraries/org/lwjgl/lwjgl-glfw/3.3.1/lwjgl-glfw-3.3.1.jar
  E:/Minecraft/MultiMC/libraries/org/lwjgl/lwjgl-jemalloc/3.3.1/lwjgl-jemalloc-3.3.1.jar
  E:/Minecraft/MultiMC/libraries/org/lwjgl/lwjgl-openal/3.3.1/lwjgl-openal-3.3.1.jar
  E:/Minecraft/MultiMC/libraries/org/lwjgl/lwjgl-opengl/3.3.1/lwjgl-opengl-3.3.1.jar
  E:/Minecraft/MultiMC/libraries/org/lwjgl/lwjgl-stb/3.3.1/lwjgl-stb-3.3.1.jar
  E:/Minecraft/MultiMC/libraries/org/lwjgl/lwjgl-tinyfd/3.3.1/lwjgl-tinyfd-3.3.1.jar
  E:/Minecraft/MultiMC/libraries/org/lwjgl/lwjgl/3.3.1/lwjgl-3.3.1.jar
  E:/Minecraft/MultiMC/libraries/com/mojang/logging/1.0.0/logging-1.0.0.jar
  E:/Minecraft/MultiMC/libraries/com/mojang/blocklist/1.0.10/blocklist-1.0.10.jar
  E:/Minecraft/MultiMC/libraries/com/mojang/patchy/2.2.10/patchy-2.2.10.jar
  E:/Minecraft/MultiMC/libraries/com/github/oshi/oshi-core/5.8.5/oshi-core-5.8.5.jar
  E:/Minecraft/MultiMC/libraries/net/java/dev/jna/jna/5.10.0/jna-5.10.0.jar
  E:/Minecraft/MultiMC/libraries/net/java/dev/jna/jna-platform/5.10.0/jna-platform-5.10.0.jar
  E:/Minecraft/MultiMC/libraries/org/slf4j/slf4j-api/1.8.0-beta4/slf4j-api-1.8.0-beta4.jar
  E:/Minecraft/MultiMC/libraries/org/apache/logging/log4j/log4j-slf4j18-impl/2.16.0/log4j-slf4j18-impl-2.16.0.jar
  E:/Minecraft/MultiMC/libraries/com/ibm/icu/icu4j/70.1/icu4j-70.1.jar
  E:/Minecraft/MultiMC/libraries/com/mojang/javabridge/1.2.24/javabridge-1.2.24.jar
  E:/Minecraft/MultiMC/libraries/net/sf/jopt-simple/jopt-simple/5.0.4/jopt-simple-5.0.4.jar
  E:/Minecraft/MultiMC/libraries/io/netty/netty-common/4.1.77.Final/netty-common-4.1.77.Final.jar
  E:/Minecraft/MultiMC/libraries/io/netty/netty-buffer/4.1.77.Final/netty-buffer-4.1.77.Final.jar
  E:/Minecraft/MultiMC/libraries/io/netty/netty-codec/4.1.77.Final/netty-codec-4.1.77.Final.jar
  E:/Minecraft/MultiMC/libraries/io/netty/netty-handler/4.1.77.Final/netty-handler-4.1.77.Final.jar
  E:/Minecraft/MultiMC/libraries/io/netty/netty-resolver/4.1.77.Final/netty-resolver-4.1.77.Final.jar
  E:/Minecraft/MultiMC/libraries/io/netty/netty-transport/4.1.77.Final/netty-transport-4.1.77.Final.jar
  E:/Minecraft/MultiMC/libraries/io/netty/netty-transport-native-unix-common/4.1.77.Final/netty-transport-native-unix-common-4.1.77.Final.jar
  E:/Minecraft/MultiMC/libraries/io/netty/netty-transport-classes-epoll/4.1.77.Final/netty-transport-classes-epoll-4.1.77.Final.jar
  E:/Minecraft/MultiMC/libraries/com/google/guava/failureaccess/1.0.1/failureaccess-1.0.1.jar
  E:/Minecraft/MultiMC/libraries/com/google/guava/guava/31.0.1-jre/guava-31.0.1-jre.jar
  E:/Minecraft/MultiMC/libraries/org/apache/commons/commons-lang3/3.12.0/commons-lang3-3.12.0.jar
  E:/Minecraft/MultiMC/libraries/commons-io/commons-io/2.11.0/commons-io-2.11.0.jar
  E:/Minecraft/MultiMC/libraries/commons-codec/commons-codec/1.15/commons-codec-1.15.jar
  E:/Minecraft/MultiMC/libraries/com/mojang/brigadier/1.0.18/brigadier-1.0.18.jar
  E:/Minecraft/MultiMC/libraries/com/mojang/datafixerupper/5.0.28/datafixerupper-5.0.28.jar
  E:/Minecraft/MultiMC/libraries/com/google/code/gson/gson/2.8.9/gson-2.8.9.jar
  E:/Minecraft/MultiMC/libraries/com/mojang/authlib/3.5.41/authlib-3.5.41.jar
  E:/Minecraft/MultiMC/libraries/org/apache/commons/commons-compress/1.21/commons-compress-1.21.jar
  E:/Minecraft/MultiMC/libraries/org/apache/httpcomponents/httpclient/4.5.13/httpclient-4.5.13.jar
  E:/Minecraft/MultiMC/libraries/commons-logging/commons-logging/1.2/commons-logging-1.2.jar
  E:/Minecraft/MultiMC/libraries/org/apache/httpcomponents/httpcore/4.4.14/httpcore-4.4.14.jar
  E:/Minecraft/MultiMC/libraries/it/unimi/dsi/fastutil/8.5.6/fastutil-8.5.6.jar
  E:/Minecraft/MultiMC/libraries/org/apache/logging/log4j/log4j-api/2.16.0/log4j-api-2.16.0.jar
  E:/Minecraft/MultiMC/libraries/org/apache/logging/log4j/log4j-core/2.16.0/log4j-core-2.16.0.jar
  E:/Minecraft/MultiMC/libraries/com/mojang/text2speech/1.13.9/text2speech-1.13.9.jar
  E:/Minecraft/MultiMC/libraries/net/fabricmc/intermediary/1.19/intermediary-1.19.jar
  E:/Minecraft/MultiMC/libraries/net/fabricmc/tiny-mappings-parser/0.3.0+build.17/tiny-mappings-parser-0.3.0+build.17.jar
  E:/Minecraft/MultiMC/libraries/net/fabricmc/sponge-mixin/0.11.4+mixin.0.8.5/sponge-mixin-0.11.4+mixin.0.8.5.jar
  E:/Minecraft/MultiMC/libraries/net/fabricmc/tiny-remapper/0.8.2/tiny-remapper-0.8.2.jar
  E:/Minecraft/MultiMC/libraries/net/fabricmc/access-widener/2.1.0/access-widener-2.1.0.jar
  E:/Minecraft/MultiMC/libraries/org/ow2/asm/asm/9.3/asm-9.3.jar
  E:/Minecraft/MultiMC/libraries/org/ow2/asm/asm-analysis/9.3/asm-analysis-9.3.jar
  E:/Minecraft/MultiMC/libraries/org/ow2/asm/asm-commons/9.3/asm-commons-9.3.jar
  E:/Minecraft/MultiMC/libraries/org/ow2/asm/asm-tree/9.3/asm-tree-9.3.jar
  E:/Minecraft/MultiMC/libraries/org/ow2/asm/asm-util/9.3/asm-util-9.3.jar
  E:/Minecraft/MultiMC/libraries/net/fabricmc/fabric-loader/0.14.8/fabric-loader-0.14.8.jar
  E:/Minecraft/MultiMC/libraries/com/mojang/minecraft/1.19/minecraft-1.19-client.jar

Native libraries:
  E:/Minecraft/MultiMC/libraries/org/lwjgl/lwjgl-glfw/3.3.1/lwjgl-glfw-3.3.1-natives-windows.jar
  E:/Minecraft/MultiMC/libraries/org/lwjgl/lwjgl-jemalloc/3.3.1/lwjgl-jemalloc-3.3.1-natives-windows.jar
  E:/Minecraft/MultiMC/libraries/org/lwjgl/lwjgl-openal/3.3.1/lwjgl-openal-3.3.1-natives-windows.jar
  E:/Minecraft/MultiMC/libraries/org/lwjgl/lwjgl-opengl/3.3.1/lwjgl-opengl-3.3.1-natives-windows.jar
  E:/Minecraft/MultiMC/libraries/org/lwjgl/lwjgl-stb/3.3.1/lwjgl-stb-3.3.1-natives-windows.jar
  E:/Minecraft/MultiMC/libraries/org/lwjgl/lwjgl-tinyfd/3.3.1/lwjgl-tinyfd-3.3.1-natives-windows.jar
  E:/Minecraft/MultiMC/libraries/org/lwjgl/lwjgl/3.3.1/lwjgl-3.3.1-natives-windows.jar

Mods:
  [✔️] fabric-api-0.57.0+1.19
  [✔️] optifabric-1.13.5
  [✔️] OptiFine_1.19_HD_U_H8_MOD
  [✔️] zergatul.freecam-1.2.0-fabric-1.19

Params:
  --username  --version 1.19 --gameDir E:/Minecraft/MultiMC/instances/1.19-Fabric/.minecraft --assetsDir E:/Minecraft/MultiMC/assets --assetIndex 1.19 --uuid  --accessToken  --userType  --versionType release

Window size: 1024 x 768

Java Arguments:
[-XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump, -Xms8192m, -Xmx8192m, -Duser.language=en]


Minecraft process ID: 4316


Using onesix launcher.

[17:37:33] [main/INFO]: Loading Minecraft 1.19 with Fabric Loader 0.14.8
[17:37:33] [main/INFO]: Loading 53 mods:
	- fabric 0.57.0+1.19
	- fabric-api-base 0.4.9+e62f51a3a9 via fabric
	- fabric-api-lookup-api-v1 1.6.7+9ff28f40a9 via fabric
	- fabric-biome-api-v1 9.0.14+b2a4a624a9 via fabric
	- fabric-blockrenderlayer-v1 1.1.18+9ff28f40a9 via fabric
	- fabric-command-api-v1 1.2.6+f71b366fa9 via fabric
	- fabric-command-api-v2 2.1.2+0d55f585a9 via fabric
	- fabric-commands-v0 0.2.23+df3654b3a9 via fabric
	- fabric-containers-v0 0.1.27+df3654b3a9 via fabric
	- fabric-content-registries-v0 3.2.0+07df213ea9 via fabric
	- fabric-convention-tags-v1 1.0.8+37622d24a9 via fabric
	- fabric-crash-report-info-v1 0.2.3+bd0a0d4aa9 via fabric
	- fabric-data-generation-api-v1 5.1.1+a680b9b4a9 via fabric
	- fabric-dimensions-v1 2.1.28+a6d2f785a9 via fabric
	- fabric-entity-events-v1 1.4.16+9ff28f40a9 via fabric
	- fabric-events-interaction-v0 0.4.26+9ff28f40a9 via fabric
	- fabric-events-lifecycle-v0 0.2.25+df3654b3a9 via fabric
	- fabric-game-rule-api-v1 1.0.19+18990361a9 via fabric
	- fabric-item-api-v1 1.5.4+767db26ea9 via fabric
	- fabric-item-groups-v0 0.3.25+9ff28f40a9 via fabric
	- fabric-key-binding-api-v1 1.0.18+9ff28f40a9 via fabric
	- fabric-keybindings-v0 0.2.16+df3654b3a9 via fabric
	- fabric-lifecycle-events-v1 2.1.0+33fbc738a9 via fabric
	- fabric-loot-api-v2 1.1.0+03a4e568a9 via fabric
	- fabric-loot-tables-v1 1.1.3+9e7660c6a9 via fabric
	- fabric-message-api-v1 1.0.1+513f4a59a9 via fabric
	- fabric-mining-level-api-v1 2.1.10+33fbc738a9 via fabric
	- fabric-models-v0 0.3.15+9ff28f40a9 via fabric
	- fabric-networking-api-v1 1.0.27+7fe97409a9 via fabric
	- fabric-networking-v0 0.3.14+df3654b3a9 via fabric
	- fabric-object-builder-api-v1 4.0.7+9ff28f40a9 via fabric
	- fabric-particles-v1 1.0.8+dc39553aa9 via fabric
	- fabric-registry-sync-v0 0.9.17+23c4cfefa9 via fabric
	- fabric-renderer-api-v1 1.0.8+9ff28f40a9 via fabric
	- fabric-renderer-indigo 0.6.9+1f92344ea9 via fabric
	- fabric-renderer-registries-v1 3.2.18+df3654b3a9 via fabric
	- fabric-rendering-data-attachment-v1 0.3.12+9ff28f40a9 via fabric
	- fabric-rendering-fluids-v1 3.0.5+9ff28f40a9 via fabric
	- fabric-rendering-v0 1.1.20+df3654b3a9 via fabric
	- fabric-rendering-v1 1.10.14+9ff28f40a9 via fabric
	- fabric-resource-conditions-api-v1 2.0.9+e62f51a3a9 via fabric
	- fabric-resource-loader-v0 0.5.5+a6d2f785a9 via fabric
	- fabric-screen-api-v1 1.0.24+f05effafa9 via fabric
	- fabric-screen-handler-api-v1 1.2.6+9ff28f40a9 via fabric
	- fabric-textures-v0 1.0.18+9ff28f40a9 via fabric
	- fabric-transfer-api-v1 2.0.9+e62f51a3a9 via fabric
	- fabric-transitive-access-wideners-v1 1.1.1+9e7660c6a9 via fabric
	- fabricloader 0.14.8
	- freecam 1.2.0
	- java 17
	- minecraft 1.19
	- mm 2.3 via optifabric
	- optifabric 1.13.5
[17:37:33] [main/INFO]: SpongePowered MIXIN Subsystem Version=0.8.5 Source=file:/E:/Minecraft/MultiMC/libraries/net/fabricmc/sponge-mixin/0.11.4+mixin.0.8.5/sponge-mixin-0.11.4+mixin.0.8.5.jar Service=Knot/Fabric Env=CLIENT
[17:37:33] [main/INFO]: Compatibility level set to JAVA_16
[17:37:33] [main/INFO]: Compatibility level set to JAVA_17
Found existing patched optifine jar, using that
[17:37:34] [main/WARN]: Configuration conflict: there is more than one oshi.properties file on the classpath
[17:37:34] [main/WARN]: Configuration conflict: there is more than one oshi.architecture.properties file on the classpath
[17:37:35] [main/ERROR]: Minecraft has crashed!
net.fabricmc.loader.impl.FormattedException: java.lang.ExceptionInInitializerError
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:463) ~[fabric-loader-0.14.8.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74) [fabric-loader-0.14.8.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23) [fabric-loader-0.14.8.jar:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
	at org.multimc.onesix.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:210) [NewLaunch.jar:?]
	at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:245) [NewLaunch.jar:?]
	at org.multimc.EntryPoint.listen(EntryPoint.java:143) [NewLaunch.jar:?]
	at org.multimc.EntryPoint.main(EntryPoint.java:34) [NewLaunch.jar:?]
Caused by: java.lang.ExceptionInInitializerError
	at net.minecraft.class_128.method_568(class_128.java:173) ~[client-intermediary.jar:?]
	at net.minecraft.class_128.method_24305(class_128.java:343) ~[client-intermediary.jar:?]
	at net.minecraft.client.main.Main.main(Main.java:152) ~[minecraft-1.19-client.jar:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:461) ~[fabric-loader-0.14.8.jar:?]
	... 10 more
Caused by: java.lang.RuntimeException: Mixin transformation of net.minecraft.class_757 failed
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.getPostMixinClassByteArray(KnotClassDelegate.java:419) ~[fabric-loader-0.14.8.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.tryLoadClass(KnotClassDelegate.java:323) ~[fabric-loader-0.14.8.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.loadClass(KnotClassDelegate.java:218) ~[fabric-loader-0.14.8.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:145) ~[fabric-loader-0.14.8.jar:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:520) ~[?:?]
	at net.optifine.reflect.Reflector.<clinit>(Reflector.java:289) ~[Optifine-mapped.jar:?]
	at net.minecraft.class_128.method_568(class_128.java:173) ~[client-intermediary.jar:?]
	at net.minecraft.class_128.method_24305(class_128.java:343) ~[client-intermediary.jar:?]
	at net.minecraft.client.main.Main.main(Main.java:152) ~[minecraft-1.19-client.jar:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:461) ~[fabric-loader-0.14.8.jar:?]
	... 10 more
Caused by: org.spongepowered.asm.mixin.transformer.throwables.MixinTransformerError: An unexpected critical error was encountered
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:392) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClass(MixinTransformer.java:234) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClassBytes(MixinTransformer.java:202) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.getPostMixinClassByteArray(KnotClassDelegate.java:414) ~[fabric-loader-0.14.8.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.tryLoadClass(KnotClassDelegate.java:323) ~[fabric-loader-0.14.8.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.loadClass(KnotClassDelegate.java:218) ~[fabric-loader-0.14.8.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:145) ~[fabric-loader-0.14.8.jar:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:520) ~[?:?]
	at net.optifine.reflect.Reflector.<clinit>(Reflector.java:289) ~[Optifine-mapped.jar:?]
	at net.minecraft.class_128.method_568(class_128.java:173) ~[client-intermediary.jar:?]
	at net.minecraft.class_128.method_24305(class_128.java:343) ~[client-intermediary.jar:?]
	at net.minecraft.client.main.Main.main(Main.java:152) ~[minecraft-1.19-client.jar:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:461) ~[fabric-loader-0.14.8.jar:?]
	... 10 more
Caused by: org.spongepowered.asm.mixin.injection.throwables.InjectionError: Critical injection failure: Callback method onRenderItemInHand(Lnet/minecraft/class_4587;Lnet/minecraft/class_4184;FLorg/spongepowered/asm/mixin/injection/callback/CallbackInfo;)V in zergatul.freecam.mixins.json:MixinGameRenderer from mod freecam failed injection check, (0/1) succeeded. Scanned 1 target(s). Using refmap zergatul.freecam-refmap.json
	at org.spongepowered.asm.mixin.injection.struct.InjectionInfo.postInject(InjectionInfo.java:468) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinTargetContext.applyInjections(MixinTargetContext.java:1385) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.applyInjections(MixinApplicatorStandard.java:1052) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.applyMixin(MixinApplicatorStandard.java:400) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.apply(MixinApplicatorStandard.java:325) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.TargetClassContext.apply(TargetClassContext.java:421) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.TargetClassContext.applyMixins(TargetClassContext.java:403) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:363) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClass(MixinTransformer.java:234) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClassBytes(MixinTransformer.java:202) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.getPostMixinClassByteArray(KnotClassDelegate.java:414) ~[fabric-loader-0.14.8.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.tryLoadClass(KnotClassDelegate.java:323) ~[fabric-loader-0.14.8.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.loadClass(KnotClassDelegate.java:218) ~[fabric-loader-0.14.8.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:145) ~[fabric-loader-0.14.8.jar:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:520) ~[?:?]
	at net.optifine.reflect.Reflector.<clinit>(Reflector.java:289) ~[Optifine-mapped.jar:?]
	at net.minecraft.class_128.method_568(class_128.java:173) ~[client-intermediary.jar:?]
	at net.minecraft.class_128.method_24305(class_128.java:343) ~[client-intermediary.jar:?]
	at net.minecraft.client.main.Main.main(Main.java:152) ~[minecraft-1.19-client.jar:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:461) ~[fabric-loader-0.14.8.jar:?]
	... 10 more
Process exited with code 1 (0x1).
Please note that usually neither the exit code, nor its description are enough to diagnose issues!
Always upload the entire log and not just the exit code.

```

**Comment by Zergatul** [comment]
I already had problems with OptiFine in 1.18 and rendering hands functionality, now it is crashing in 1.19...
I never used OptiFine with Fabric, I just need optifabric to test this?

**Comment by mubbery** [comment]
OptiFabric is just a loader for Optifine so you need both bits.

https://www.curseforge.com/minecraft/mc-mods/optifabric

**Comment by Zergatul** [comment]
This is fixed, please check version 1.2.1. But render hands option will not work with OptiFine.

**Comment by mubbery** [comment]
Confirmed working now. Many thanks.
